cmd/skaffold/app/cmd/deploy.go

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright 2018 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"io"
+	"io/ioutil"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
+	"github.com/google/go-containerregistry/name"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	images []string
+)
+
+// NewCmdDeploy describes the CLI command to deploy artifacts.
+func NewCmdDeploy(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deploy",
+		Short: "Deploys the artifacts",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runDeploy(out, filename)
+		},
+	}
+	AddRunDevFlags(cmd)
+	cmd.Flags().StringSliceVar(&images, "images", nil, "A list of images to deploy")
```

Comment:
Any reason not to make these args instead of a flag?

Reply:
If they are args we can't have any more nested subcommands. Not really planning any now but thats been confusing in the past.